_includes/components/search-results/SearchResult/partials/buildNotice.jsx

```diff
@@ -0,0 +1,61 @@
+import { h } from 'preact';
+
+
+const calcNoticeObj = (amount, error) => {
+  if (error) {
+    return {
+      size: 'is-1of1',
+      title: 'Something went wrong.',
+      text: 'Please try again at a later point.',
+    };
+  }
+
+  switch (amount) {
```

Comment:
I like this but I don't see it working
![vulekamali-improved-search-missing-help-result](https://user-images.githubusercontent.com/235801/39923619-b83597da-5524-11e8-9fb4-1d225b6dd5b2.png)